src/Entity/Book.php

```diff
@@ -55,10 +55,133 @@ public function __construct()
         $this->physicalBooks = new ArrayCollection();
         $this->reviews = new ArrayCollection();
         $this->borrows = new ArrayCollection();
+        $this->id = Uuid::v4();
     }
 
     public function getId(): Uuid
     {
         return $this->id;
     }
+
+    
+
+    /**
+     * Get the value of writerFirstname
+     */ 
+    public function getWriterFirstname()
+    {
+        return $this->writerFirstname;
+    }
+
+    /**
+     * Set the value of writerFirstname
+     *
+     * @return  self
+     */ 
+    public function setWriterFirstname($writerFirstname)
+    {
+        $this->writerFirstname = $writerFirstname;
+
+        return $this;
+    }
+
+    /**
+     * Get the value of writerLastname
+     */ 
+    public function getWriterLastname()
```

Comment:
Could you please add type hints for all arguments and return types to all methods?
This will help ensuring the correctness of the program.